File: comms/email/delete-email/delete-email.ts
import { TwilioClient } from "twilio-comms";

async function main() {
    const client = new TwilioClient({
        accountId: "TWILIO_ACCOUNT_SID",
        authToken: "TWILIO_AUTH_TOKEN",
    });
    await client.emails.delete("comms_email_01h9krwprkeee8fzqspvwy6nq8");
}
main();

File: comms/email/delete-email/delete-email.rb
require "twiliocomms"

client = TwilioComms::Client.new(
  account_id: "TWILIO_ACCOUNT_SID",
  auth_token: "TWILIO_AUTH_TOKEN"
)

client.emails.delete(email_id: "comms_email_01h9krwprkeee8fzqspvwy6nq8")
